packages/protocol/contracts/identity/FederatedAttestations.sol

```diff
@@ -112,29 +112,96 @@ contract FederatedAttestations is
   }
```

Comment:
technically `pure` functions should be put at the end of their group of functions (ie. after the other `external` functions), but feels weird to move `getVersionNumber` down

Reply:
I'd say we put it right under the constructor since that's what we're doing in `GrandaMento.sol` and `MetaTransactionWallet.sol`, will move it